wasm-runtime/test/wasm-apps/opensnoop.c

```diff
@@ -11,8 +11,12 @@
 /// @return 0 on success, -1 on failure, the eBPF program will be terminated in
 /// failure case
 int
-bpf_main(char *env_json, int str_len)
+main(int argc, char **args)
 {
+    if (argc != 3)
+        printf("error: the number of parameters passed is not equal to 2.");
+    int str_len = atoi(args[2]);
```

Comment:
This code is not necessary as env_json and str_len are not actually used (I will remove it later).